ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js

```diff
@@ -610,10 +612,23 @@ module.exports = function OpenlayersMap(
     showMultiLineShape: function(locationModel) {
       let lineObject = locationModel
         .get('multiline')
-        .map(line => line.map(coords => convertPointCoordinate(coords)))
+        .map(line => line.map(coords => unconvertPointCoordinate(coords)))
+
+      const bufferedWidth =
+        DistanceUtils.getDistanceInMeters(
+          locationModel.get('lineWidth'),
+          locationModel.get('lineUnits')
+        ) || 1
```

Comment:
✏️ 1 is being used as a default/fallback here from what I can tell. Should we make this a constant?

Reply:
Yep, that's the same fallback used for polygons and lines as well. Added the constant 👍